src/main/java/views/LeagueSimulatorPane.java

```diff
@@ -0,0 +1,220 @@
+    final ComboBox<String> comboBox7 = new ComboBox<>(FXCollections.observableList(teamNames));
+    final ComboBox<String> comboBox8 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox4 = new VBox();
+
+    private HBox matches3and4 = new HBox();
+
+    private Label match5 = new Label("Match 5:");
+    final ComboBox<String> comboBox9 = new ComboBox<>(FXCollections.observableList(teamNames));
+    final ComboBox<String> comboBox10 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox5 = new VBox();
+
+    private Label match6 = new Label("Match 6:");
+    final ComboBox<String> comboBox11 = new ComboBox<>(FXCollections.observableList(teamNames));
+    final ComboBox<String> comboBox12 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox6 = new VBox();
+
+    private HBox matches5and6 = new HBox();
+
+    private Label match7 = new Label("Match 7:");
+    final ComboBox<String> comboBox13 = new ComboBox<>(FXCollections.observableList(teamNames));
+    final ComboBox<String> comboBox14 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox7 = new VBox();
+
+    private Label match8 = new Label("Match 8:");
+    final ComboBox<String> comboBox15 = new ComboBox<>(FXCollections.observableList(teamNames));
+    final ComboBox<String> comboBox16 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox8 = new VBox();
+
+    private HBox matches7and8 = new HBox();
+
+    public LeagueSimulatorPane() {
+        connectComponents();
+        styleComponents();
+        setCallbacks();
+    }
+
+    @Override
+    void connectComponents() {
+
+        leftContainer.getChildren().addAll(title, matchesContainer, playButton, returnButton);
+
+        matchBox1.getChildren().addAll(match1, comboBox1, comboBox2);
+        matchBox2.getChildren().addAll(match2, comboBox3, comboBox4);
+        matchBox3.getChildren().addAll(match3, comboBox5, comboBox6);
+        matchBox4.getChildren().addAll(match4, comboBox7, comboBox8);
+        matchBox5.getChildren().addAll(match5, comboBox9, comboBox10);
+        matchBox6.getChildren().addAll(match6, comboBox11, comboBox12);
+        matchBox7.getChildren().addAll(match7, comboBox13, comboBox14);
+        matchBox8.getChildren().addAll(match8, comboBox15, comboBox16);
+
+        matches1and2.getChildren().addAll(matchBox1,matchBox2);
+        matches3and4.getChildren().addAll(matchBox3,matchBox4);
+        matches5and6.getChildren().addAll(matchBox5,matchBox6);
+        matches7and8.getChildren().addAll(matchBox7,matchBox8);
+
+        matchesContainer.getChildren().addAll(matches1and2,matches3and4,matches5and6,matches7and8);
+
+        this.setLeft(leftContainer);
+        this.setCenter(centerContainer);
+    }
+
+    @Override
+    void styleComponents() {
+        leftContainer.setStyle(" -fx-padding: 20 20 20 20;\n" +
```

Comment:
I'd indent these a bit more such that the quotes all start at the same column! Why is there some whitespace following the quotes? Not sure if it matters but could be removed if it doesn't.